### ISSUE TITLE
Add some features to JS yii 

### ISSUE THREAD

**Issue by alexandernst** [yiisoft/yii2#8673]
Currently `yii` (the JS) can retrieve the csrf and a few more things. It would be nice if we could get the current language or the available languages (the languages that have been defined in the i18n configuration). Maybe even some more things.

EDIT: Suggestions
- `yii.lang` - Current language (as reported by `Yii::$app->language`)
- `yii.langs` - All available languages (all languages configured in the i18n section of the config file)
- `yii.renderPartial(selector, url, [data])` - a helper method that will call `url` and make the controller render a partial view and replace the HTML of `selector`. An optional `data` argument could be passed, so the controller can receive some data.
- `yii.is_debug` - a bool indicating if Yii2 is running in debug mode. (useful for tweaking `console.log` to print only when in debug mode)

**Comment by unclead** [comment]
i agreed that possibility of getting current language can be usefull. Nowadays i have to pass it to js from view

```
var common = {
    options: {},
    ...
    init: function(options) {
        var self = common;
        self.options = options;
        self.setLanguage(self.options.language);
    },

    setLanguage: function(language) {
        yii.lang = language;
        bootbox.setDefaults({locale: yii.lang});
    },
```

**Comment by alexandernst** [comment]
I added another suggestion to my initial post. I'll keep updating it whenever I get a new idea. You can then decide which one should be implemented and which one not.

**Comment by samdark** [comment]
Won't be implemented in 2.0.
